evi-modal-clm/modal_app.py

```diff
@@ -0,0 +1,15 @@
+from modal import Image, App, asgi_app
```

Comment:
maybe it makes sense to put this in a folder for modal similar to the folder for cdk?